### PR TITLE
feat: support mako build

### PR DESCRIPTION
**Pull request by Jinbao1001**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    support mako build       |
| 🇨🇳 Chinese |    接入 mako 构建       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx [hunk 1]
```diff
@@ -142,10 +142,17 @@ interface ChangelogInfo {
...
+    lang === 'cn'
+      ? {
+          key: 'component-changelog-cn',
+          request: () => import('../../../preset/components-changelog-cn.json'),
+        }
+      : {
+          key: 'component-changelog-en',
+          request: () => import('../../../preset/components-changelog-en.json'),
+        },
+  );
```
Comment (MadCcc) [inline .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:155]: 这里改了是为啥

**Comment by MadCcc** [comment]
@SocketSecurity ignore npm/esbuild@0.18.20
@SocketSecurity ignore npm/core-js-pure@3.37.1

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx (hunk 1 above)
Comment (Jinbao1001) [reply .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:155]: mako 对这种 case 解析暂时有问题, 先改了试用下, mako会同步跟进修复.

**Comment by MadCcc** [comment]
@SocketSecurity ignore npm/sharp@0.33.4
@SocketSecurity ignore npm/puppeteer@22.10.0
@SocketSecurity ignore npm/fsevents@1.2.13
@SocketSecurity ignore npm/core-js@3.34.0

**Comment by zombieJ** [comment]
@SocketSecurity ignore npm/@ant-design/tools@18.0.2
@SocketSecurity ignore npm/es5-ext@0.10.64

File: package.json [hunk 2]
```diff
@@ -354,6 +354,11 @@
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
+  "overrides": {
+      "dumi-plugin-color-chunk": {
+          "dumi": "^2.4.0-alpha.4"
+      }
+  },
```
Comment (afc163) [inline package.json:361]:
```suggestion
  "overrides": {
    "dumi-plugin-color-chunk": {
      "dumi": "^2.4.0-alpha.4"
    }
  },
```

**Comment by afc163** [comment]
@SocketSecurity ignore npm/esbuild@0.21.4
@SocketSecurity ignore npm/@biomejs/biome@1.8.0

File: mako.config.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "optimization": {
+    "skipModules": false,
+    "concatenateModules": false
```
Comment (afc163) [inline mako.config.json:4]: 这两个属性有 bug？

Comment (Jinbao1001) [reply mako.config.json:4]: 构建产物有问题, 先关了, 影响一点产物size, 等修复了再放开.

**Comment by MadCcc** [comment]
@SocketSecurity ignore npm/@swc/core@1.4.2
@SocketSecurity ignore npm/@fortawesome/free-regular-svg-icons@6.5.2
@SocketSecurity ignore npm/@fortawesome/free-solid-svg-icons@6.5.2
@SocketSecurity ignore npm/@fortawesome/fontawesome-common-types@6.5.2

**Comment by zombieJ** [comment]
@SocketSecurity ignore npm/@biomejs/biome@1.8.1

**Review by MadCcc — APPROVED** [review]

**Comment by MadCcc** [comment]
非常好 PR，爱来自 antd

**Comment by crazyair** [comment]
想问下，为啥要升级 dayjs 版本

**Pull request merged**